rotkehlchen/chain/gnosis/modules/sdai/decoder.py

```diff
@@ -0,0 +1,114 @@
+from typing import TYPE_CHECKING, Any
+
+from rotkehlchen.chain.ethereum.utils import token_normalized_value_decimals
+from rotkehlchen.chain.evm.constants import \
+    DEFAULT_TOKEN_DECIMALS  # all tokens in this decoder use default(18)
```

Comment:
Never use `\`. Never. We don't use this in the codebase. To avoid the newline rule I told you even in the PR review use noqa: E501.

```suggestion
from rotkehlchen.chain.evm.constants import DEFAULT_TOKEN_DECIMALS  # all tokens in this decoder use default(18)  # noqa: E501
```